src/maxdiffusion/generate.py

```diff
@@ -71,7 +71,12 @@ def loop_body(step, args, model, pipeline, prompt_embeds, guidance_scale, guidan
     noise_pred = noise_pred_uncond + guidance_scale * (noise_prediction_text - noise_pred_uncond)
 
     # Based on 3.4. in https://arxiv.org/pdf/2305.08891.pdf
-    noise_pred = rescale_noise_cfg(noise_pred, noise_prediction_text, guidance_rescale=guidance_rescale)
```

Comment:
nit: 
Adding a comment around what is being done
# This technique aims to improve image quality by adjusting the noise prediction.

Reply:
Added comments to explain what this does.